tools/gfx/persistent-shader-cache.cpp

```diff
@@ -13,7 +13,7 @@ PersistentShaderCache::PersistentShaderCache(const Desc& desc)
 {
     entryCountLimit = desc.entryCountLimit;
```

Comment:
Just make `desc` a member and then initialize that member directly with the parameter `desc` (rename the parameter to `inDesc` here to avoid name conflicts).

Reply:
Will move the default values all the way up to the ShaderCacheDesc in slang-gfx.h in the other PR and have them be passed in from there.